### PR TITLE
rust 1.44.0

### PR DESCRIPTION
**Pull request by samford**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` ~~(after doing `brew install <formula>`~~)?

-----

It takes forever to build this from source, so I usually just let it run on CI to see what happens (being careful to avoid simple mistakes beforehand).

### CODE REVIEW

**Comment by samford** [comment]
The only failure was a SHA256 mismatch in `oxipng`:

```
==> brew install --build-from-source oxipng
==> FAILED
==> Downloading https://github.com/shssoichiro/oxipng/archive/v3.0.0.tar.gz
==> Downloading from https://codeload.github.com/shssoichiro/oxipng/tar.gz/v3.0.0
Error: SHA256 mismatch
Expected: 00edbe20dc18f80b00f96dc6389b15c90c9f6068b207d3b282cef3c75fed0fe9
  Actual: fd7584299375a630322b152878756297b0083492b7e4b9b17ae9978662e8c36a
 Archive: /Users/brew/Library/Caches/Homebrew/downloads/5cd7f138a1157668f34d10c4e9bcef35e2a0d1ee4039c5a47832bfd2d173c0db--oxipng-3.0.0.tar.gz
To retry an incomplete download, remove the file above.
```

If no one gets to this before me, I'll try to take care of it later today.

File: Formula/rust.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@ class Rust < Formula
   homepage "https://www.rust-lang.org/"
 
   stable do
-    url "https://static.rust-lang.org/dist/rustc-1.43.1-src.tar.gz"
-    sha256 "cde177b4a8c687da96f20de27630a1eb55c9d146a15e4c900d5c31cd3c3ac41d"
+    url "https://static.rust-lang.org/dist/rustc-1.44.0-src.tar.gz"
+    sha256 "bf2df62317e533e84167c5bc7d4351a99fdab1f9cd6e6ba09f51996ad8561100"
 
     resource "cargo" do
       url "https://github.com/rust-lang/cargo.git",
```
Comment (Bo98) [inline Formula/rust.rb:10]: This resource should also be bumped to 0.45.0.

Comment (samford) [reply Formula/rust.rb:10]: Augh, the Cargo 0.45.0 release didn't exist yet when I created the PR, unfortunately. I'll have to remember to wait a bit in the future. Thanks for the catch.

I'm going to kill this CI run for now and I'll restart it once the `oxipng` issue is resolved.

**Comment by SMillerDev** [comment]
I cancelled the remaining catalina build since it probably won't have a different error and had been running for 24 hours.

---

Geeqie website is down, we could use https://web.archive.org/web/20191026105315/http://www.geeqie.org/geeqie-1.5.1.tar.xz or remove the formula.

**Comment by samford** [comment]
> Geeqie website is down

It seems to be up now and `brew audit --strict --online geeqie` works again. Earlier I was able to access the website and tarball in the browser but I was still getting the `not reachable` error in `brew audit`. Seems to be fine now but that may be temporary.

---

Do we need to re-run this once `geeqie` is fine or are we fine merging this as it is?

**Comment by SMillerDev** [comment]
I'd report it upstream at the very least. We could try downloading a tarbal from git `http://www.geeqie.org/cgi-bin/gitweb.cgi?p=geeqie.git;a=snapshot;h=refs/tags/v1.5.1;sf=tgz`

**Comment by Bo98** [comment]
The `geeqie` issue is known: https://github.com/BestImageViewer/geeqie/issues/774

**Comment by mowgli** [comment]
Geeqie website should be fully available again now.

**Comment by samford** [comment]
Seeing as the `geeqie` issue is resolved, I'm going to go ahead and merge this.

**Pull request closed without merging**